### PR TITLE
feat(enrichment): get nuts code from coordinates - EUBFR-115

### PR DESCRIPTION
**Pull request by degliwe**
@kalinchernev: one can take AGRI as an example, as it provides [coordinates, but no NUTS info

TBD: `nuts` field and types in terms of elasticsearch, flow, etc.

### CODE REVIEW

File: services/enrichment/manager/src/lib/checks.js [hunk 1]
```diff
@@ -5,16 +5,14 @@ export const canLocationsBeEnriched = locations =>
...
-
-    // Country code is empty but centroid is provided
-    const needsCountryCode = !loc.country_code && loc.centroid;
-
-    if (needsCountryCode) return true;
-
-    return false;
+    // Centroid is empty but country code is provided OR
+    // Country code is empty but centroid is provided OR
+    // there are no NUTS code provided
```
Comment (yhuard) [inline services/enrichment/manager/src/lib/checks.js]: `there are no NUTS code` is not enough to tell if the record can be enriched or not

File: services/value-store/projects/src/lib/SaveStream.js [hunk 2]
```diff
@@ -14,19 +14,16 @@ export default class SaveStream extends stream.Writable {
     // Manually calculate the ID
     // md5 hash of computed_key + project_id
     const { computed_key: computedKey, project_id: projectId } = chunk;
+
+    const body = Object.assign({}, chunk, {
+      enriched: false,
```
Comment (yhuard) [inline services/value-store/projects/src/lib/SaveStream.js]: 👍

File: services/ingestion/helpers/getCountryCode.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+// Helper to get country code based on few exceptions
```
Comment (yhuard) [inline services/ingestion/helpers/getCountryCode.js]: To be clear, here we make the conversion NUTS0 -> ISO_3166-2 😉

File: services/ingestion/etl/inforegio/xml/test/unit/lib/__snapshots__/transform.spec.js.snap [hunk 4]
```diff
@@ -46,9 +46,16 @@ Object {
     Object {
       "address": "",
       "centroid": null,
-      "country_code": "EL",
+      "country_code": "GR",
       "location": null,
-      "nuts2": "EL0",
+      "nuts": Array [
+        Object {
+          "code": "EL0",
```
Comment (yhuard) [inline services/ingestion/etl/inforegio/xml/test/unit/lib/__snapshots__/transform.spec.js.snap]: `EL0` doesn't exist 😬 `EL3`, `EL4`, `EL5` and `EL6` do. By the way, the level should be `1`, not `2`

File: services/ingestion/helpers/getCountryCode.js (hunk 3 above)
Comment (kalinchernev) [reply services/ingestion/helpers/getCountryCode.js]: Added these details :)

File: services/value-store/projects/src/lib/SaveStream.js (hunk 2 above)
Comment (kalinchernev) [reply services/value-store/projects/src/lib/SaveStream.js]: I actually removed this change as I applied it wrongly on the project and not the location. Then, I decided to not add a false flag at all, as when needed, flag becomes `true` and it's enough to check for truthy values where needed.

File: services/ingestion/etl/inforegio/xml/src/lib/transform.js [hunk 5]
```diff
@@ -170,8 +170,8 @@ const getLocations = record => {
         {
           code: checkData(record.Project_NUTS2_code),
           name: '',
-          level: 2,
-          year: '',
+          level: null,
```
Comment (yhuard) [inline services/ingestion/etl/inforegio/xml/src/lib/transform.js]: why don't you get the level from the code?

File: services/value-store/projects/src/lib/SaveStream.js (hunk 2 above)
Comment (yhuard) [reply services/value-store/projects/src/lib/SaveStream.js]: fine for me

**Pull request merged**